resources/definitions/prusa_i3_mk2.def.json

```diff
@@ -0,0 +1,53 @@
+{
+    "id": "prusa_i3_mk2",
+    "version": 2,
+    "name": "Prusa i3 Mk2",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Apsu",
+        "manufacturer": "Prusa Research",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "icon": "icon_ultimaker2",
+        "platform": "prusai3_platform.stl",
+        "has_materials": true
+    },
+
+    "overrides": {
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 250 },
+        "machine_height": { "default_value": 200 },
+        "machine_depth": { "default_value": 210 },
+        "machine_center_is_zero": { "default_value": false },
+        "material_diameter": { "default_value": 1.75 },
+        "material_bed_temperature": { "default_value": 55 },
+        "machine_nozzle_size": { "default_value": 0.4 },
+        "layer_height": { "default_value": 0.1 },
+        "layer_height_0": { "default_value": 0.15 },
+        "retraction_amount": { "default_value": 0.8 },
+        "retraction_speed": { "default_value": 35 },
+        "retraction_retract_speed": { "default_value": 35 },
+        "retraction_prime_speed": { "default_value": 35 },
+        "adhesion_type": { "default_value": "skirt" },
+        "machine_nozzle_heat_up_speed": { "default_value": 2 },
+        "machine_nozzle_cool_down_speed": { "default_value": 2 },
+        "machine_head_with_fans_polygon": { "default_value": [[0,0],[0,0],[0,0],[0,0]] },
+        "gantry_height": { "default_value": 0 },
```

Comment:
Sure, and that makes sense, but I suspect it needs to know more than just the height in order to avoid hitting the previous parts. I can take a stab at putting in the right values for the gantry and extruder and attempt to test with a couple of very close tall parts in one-at-a-time mode, or similar.

I figure the dimensions are about the extruder assembly bounding box, and the gantry height is about the lowest rod carrying the extruder, so neither bumps into a previous item. Is that correct?